zeppelin-web/src/components/navbar/navbar.html

```diff
@@ -65,7 +65,7 @@
         <!--TODO(bzz): move to Typeahead https://angular-ui.github.io/bootstrap  -->
           <form role="search"
             style="width: 300px; display: inline-block; margin: 0 10px"
-             ng-submit="search()">
+             ng-submit="search(searchTerm)">
```

Comment:
i definitely like passing the argument to `search` instead of taking it from the `$scope` 👍